src/subcommand/wallet/inscribe/batch.rs

```diff
@@ -568,13 +569,22 @@ impl Batchfile {
 
   pub(crate) fn inscriptions(
     &self,
+    client: &Client,
     chain: Chain,
     parent_value: Option<u64>,
     metadata: Option<Vec<u8>>,
     postage: Amount,
     compress: bool,
-  ) -> Result<Vec<Inscription>> {
+  ) -> Result<(Vec<Inscription>, Vec<Address>)> {
     assert!(!self.inscriptions.is_empty());
+    assert!(
```

Comment:
We should return an anyhow error here, since this is an error that the user could trigger.